nixos/modules/services/hardware/asusd.nix

```diff
@@ -19,55 +17,55 @@ in
       };
 
       animeConfig = lib.mkOption {
-        type = json.type;
-        default = { };
+        type = lib.types.nullOr lib.types.str;
+        default = null;
         description = lib.mdDoc ''
-          The content of /etc/asusd/anime.conf.
+          The content of /etc/asusd/anime.ront.
```

Comment:
Also typo here

Reply:
fixed

Reply:
Thanks! Can you also add the release note? Somewhere around https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md#backward-incompatibilities-sec-release-2305-incompatibilities